recipe/patches/0005-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -1,34 +1,27 @@
-From e0393030155231b8579c523bc0c3111a5ea1928b Mon Sep 17 00:00:00 2001
+From d8fb07467ed7b3cdf99d46ffcd86a98c6e86828f Mon Sep 17 00:00:00 2001
 From: Kai Fricke <kai@anyscale.com>
 Date: Fri, 3 Dec 2021 10:55:23 -0800
-Subject: [PATCH 3/5] Remove all dependencies from setup.py
+Subject: [PATCH 5/5] Remove all dependencies from setup.py
 
 Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
 Signed-off-by: Matti Picus <matti.picus@gmail.com>
 ---
- python/setup.py | 108 +++++-------------------------------------------
- 1 file changed, 11 insertions(+), 97 deletions(-)
+ python/setup.py | 98 ++++---------------------------------------------
+ 1 file changed, 8 insertions(+), 90 deletions(-)
 
 diff --git a/python/setup.py b/python/setup.py
-index 8d988292f0..6d1f3e7eb1 100644
+index 9eef7cca46..74eb56681a 100644
 --- a/python/setup.py
 +++ b/python/setup.py
-@@ -225,82 +225,21 @@ ray_files += [
+@@ -225,76 +225,18 @@ ray_files += [
  # also update the matching section of requirements/requirements.txt
  # in this directory
  if setup_spec.type == SetupType.RAY:
--    if sys.version_info >= (3, 7):
--        pandas_dep = "pandas >= 1.3"
--        numpy_dep = "numpy >= 1.20"
--    else:
--        # Pandas dropped python 3.6 support in 1.2.
--        pandas_dep = "pandas >= 1.0.5"
--        # NumPy dropped python 3.6 support in 1.20.
--        numpy_dep = "numpy >= 1.19"
--    if sys.version_info >= (3, 7) and sys.platform != "win32":
+-    pandas_dep = "pandas >= 1.3"
+-    numpy_dep = "numpy >= 1.20"
+-    if sys.platform != "win32":
 -        pyarrow_dep = "pyarrow >= 6.0.1"
 -    else:
--        # pyarrow dropped python 3.6 support in 7.0.0.
 -        # Serialization workaround for pyarrow 7.0.0+ doesn't work for Windows.
 -        pyarrow_dep = "pyarrow >= 6.0.1, < 7.0.0"
```

Comment:
Actually, I note that this cap has existed for a long time, but we never reflected that in our metadata, and received no user complaints to the best of my knowledge (despite thousands of downloads on windows).

So I'm now even more convinced that (re-)adding the cap is not worth it. Yes that upstream bug should get fixed, but we should not cripple the windows packages for that.

Reply:
Ok, I agree. Would you like to remove the cap?